README.md

```diff
@@ -240,6 +240,10 @@ Then, by typing `cbv.actionType.` as a parameter of the `setAction` method of th
 all the attributes associated to this particular CBV.
 The list of all CBVs and the respective attributes can be viewed in ./src/cbv/cbv.js.
 
+### Custom vocabulary attributes
+
+You can import all the `vtype` for the `VocabularyElement` type.
```

Comment:
The SDK enables you to easily search among all default `vtypes` of as defined in the GS1 Web Vocabulary. 
By typing `cbv.actionType.`, you will be displayed all the possible vtypes.